### PR TITLE
Removing expensive logging from v2 RowMerger.

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/v2/RowMerger.java [hunk 1]
```diff
@@ -108,24 +108,20 @@ void handleLastScannedRowKey(ByteString lastScannedRowKey) {
...
-      void validateChunk(RowInProgress rowInProgess, ByteString previousKey, CellChunk newChunk) {
-        Preconditions.checkArgument(
-            rowInProgess == null, "A new row cannot have existing state: " + newChunk);
-        Preconditions.checkArgument(
-            newChunk.getRowStatusCase() != RowStatusCase.RESET_ROW,
-            "A new row cannot be reset: " + newChunk);
-        Preconditions.checkArgument(
-            !newChunk.getRowKey().isEmpty(), "A row key must be set: " + newChunk);
-        Preconditions.checkArgument(newChunk.hasFamilyName(), "A family must be set: " + newChunk);
+          void validateChunk(RowInProgress rowInProgess, ByteString previousKey,
```
Comment (garye) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/v2/RowMerger.java]: indented 2 extra spaces

**Comment by garye** [comment]
LGTM

Makes sense not to print out the entire chunk (would the entire value get printed?) but there's a varargs version of checkArgument that uses a format string in case that solves your problem...

File: bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/v2/RowMerger.java (hunk 1 above)
Comment (sduskis) [inline bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/v2/RowMerger.java]: Fixed the indentation.  It was related to eclipse auto formatting.

**Comment by sduskis** [comment]
I updated RowMerger validation to use varags.  Thanks for the tip.

**Pull request merged**